### ISSUE TITLE
Why does `training_window` in `dfs()` not support 'month'?

### ISSUE THREAD

**Issue by rightx2** [FeatureLabs/featuretools#719]
### Why does `training_window` not support month?
(replace this text with your issue)

 

#### Bug/Feature Request Description

`ft.dfs` is not support only `month`!!

I checked it out by printing some variables in `entityset/timedelta.py`

```
In [2]: self._readable_units
Out[2]: 
{'ms': 'Milliseconds',
 's': 'Seconds',
 'h': 'Hours',
 'm': 'Minutes',
 'd': 'Days',
 'o': 'Observations',
 'w': 'Weeks',
 'Y': 'Years',
 'u': 'Units'}

In [4]: self._readable_to_unit
Out[4]: 
{'milliseconds': 'ms',
 'seconds': 's',
 'hours': 'h',
 'minutes': 'm',
 'days': 'd',
 'observations': 'o',
 'weeks': 'w',
 'years': 'Y',
 'units': 'u'}
``` 

Any reason for that?

**Comment by jeremyliweishih** [comment]
Hello @rightx2!

Could you please check what version of Featuretools you are on? Relative units such as `month` was added in Featuretools v0.10.0. If you're on a lower version you could upgrade with `pip install -U featuretools`. Thanks and let me know if you have any other questions.

**Comment by rightx2** [comment]
@jeremyliweishih  I've upgraded lib to 0.10.1 but still doesn't work

What I've tried:

1. `training_window="3 mo", training_window="3 months", training_window="3 month"`

-> It gave me an error: `AssertionError: training window must be an absolute Timedelta`

2. `training_window=pd.Timedelta(3, "M")`

-> It runs ok but feature_matrix generated was wrong. It seemed like it didn't do any calculation....


3. `training_window=relativedelta(months=3)`

-> errors: `ValueError: Unable to parse timedelta: relativedelta(months=+3)`


Notice: If I removed `training_window` args, it worked like a charm based on `cutoff_time`.
